src/functionalTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/ProcessedEnvelopeMessageHandlingTest.java

```diff
@@ -71,9 +74,15 @@ public void should_complete_envelope_referenced_by_queue_message() throws Except
         EnvelopeResponse updatedEnvelope = getEnvelope(zipFilename);
         assertThat(updatedEnvelope.getScannableItems()).hasSize(2);
         assertThat(updatedEnvelope.getScannableItems()).allMatch(item -> item.ocrData == null);
-        assertThat(updatedEnvelope.getCcdId()).isEqualTo(ccdId);
+
+        if (fluxFuncTest) {
+            assertThat(updatedEnvelope.getCcdId()).isNotEmpty();
```

Comment:
should this not be `EXCEPTION_RECORD` ?

Reply:
it is the id of whatever ccd creates with our request.